### PR TITLE
add trans_prob

### PR DESCRIPTION
**Pull request by jin-s13**

### CODE REVIEW

File: mmpose/datasets/pipelines/top_down_transform.py [hunk 1]
```diff
@@ -665,16 +665,20 @@ class TopDownRandomTranslation:
...
 
     def __call__(self, results):
         """Perform data augmentation with random translation."""
         center = results['center']
         scale = results['scale']
-        # reference bbox size is [200, 200] pixels
-        center += self.trans_factor * (2 * np.random.rand(2) - 1) * scale * 200
+        if np.random.rand() <= self.trans_prob:
+            # reference bbox size is [200, 200] pixels
+            center += self.trans_factor * (2 * np.random.rand(2) -
```
Comment (innerlee) [inline mmpose/datasets/pipelines/top_down_transform.py:681]: `np.random.uniform(-1, 1, size=2)`

**Review by innerlee — APPROVED** [review]

**Comment by ly015** [comment]
Do we need to make reference bbox size an argument with a default value (200, 200)?

**Comment by jin-s13** [comment]
> Do we need to make reference bbox size an argument with a default value (200, 200)?

Not necessary for now, I think. And this involves a lot of code changes...

**Comment by innerlee** [comment]
Yeah, currently our tactic is to add comments on every appearance of 200

**Pull request merged**